### ISSUE TITLE
How to make resizable ol.control.Overlay menu in ol-ext?

### ISSUE THREAD

**Issue by erginduran** [Viglino/ol-ext#363]
[Question/Request]

I am using ol.control.Overlay menu in my OL3 project. It slides very good on the map but i want to make that div/menu resizable (Jquery ui-resizable). 
But it doesn't work with ol-overlay. When i set:
`$('#menu').resizable() or $("#menu").parent().resizable // to catch parent div which has been created `by ol.control.overlay. it add a ui-resizable class to div
```
.ui-resizable {
    position:relative; // but ol-overlay class blocks this on css.
}
```
if i force to use "position:relative !important;" in custom menu class, menu doesn't appear on the screen.

Basically how to make overlay menu resizable.
[ol-ext: ol.control.Overlay Menu Example Link](https://viglino.github.io/ol-ext/examples/control/map.control.overlay.menu.html)

Thanks in advance! @viglino

**Comment by Viglino** [comment]
You have to make the menu container resizable:
```javascript
$("#menu").parent().resizable();
```
You also have to set the position to absolute and prevent transition on resize (just let transform on it):
```css
#map .ui-resizable {
  position: absolute;
}
#map .ol-overlay.ol-visible {
  -webkit-transition: transform 0.5s;
  transition: transform 0.5s;
}
```
See example online: https://codepen.io/viglino/pen/oNNgXYJ

**Comment by erginduran** [comment]
I have tried  that make parent div resizable with parent(). But these line makes sense for me! 
`#map .ui-resizable {
  position: absolute;
}
#map .ol-overlay.ol-visible {
  -webkit-transition: transform 0.5s;
  transition: transform 0.5s;
}`

Thanks a lot! @Viglino You're best one!
